src/app/hooks/use-profile-synchronizer.ts

```diff
@@ -231,7 +231,7 @@ export const useProfileSynchronizer = ({ onProfileRestoreError }: ProfileSynchro
 				await profile.sync();
 				await persist();
 
-				runAll();
+				await runAll();
```

Comment:
Hey @w3ea do you see any issues if we don't await for `runAll`? Asking because not awaiting for all tasks to finish, in order profile status to synced, was intentional for performance.

Reply:
@goga-m i didn't know about this, thanks
i added a comment for that, if you know better comment just tell me or feel free to add it :wink: